src/garage/experiment/__init__.py

```diff
@@ -3,15 +3,11 @@
 from garage.experiment.experiment import to_local_command
 from garage.experiment.experiment import variant
 from garage.experiment.experiment import VariantGenerator
-from garage.experiment.snapshotter import Snapshotter
-
-# LocalRunner needs snapshotter to be imported, so we have to use a strange
-# import order here
-snapshotter = Snapshotter()
+from garage.experiment.snapshotter import SnapshotConfig, Snapshotter
 
 from garage.experiment.local_tf_runner import LocalRunner  # noqa: I100,E402,E501,I202 pylint: disable=wrong-import-position
```

Comment:
please remove all these linting exceptions and attempt to place this at the correct position (you may need to re-apply some)